content/applications/inventory_and_mrp/manufacturing/management/split_merge.rst

```diff
@@ -0,0 +1,82 @@
+====================================
+Split and merge manufacturing orders
+====================================
```

Comment:
```suggestion
======================================
Manufacturing orders (split and merge)
======================================
```
purely just a minor suggestion - not a requirement - it's only to adhere to JCS' desire to never start doc H1s with a verb. Totally your call on this one.